src/Microsoft.IIS.Administration.WebServer.Info/InfoHelper.cs

```diff
@@ -18,13 +18,23 @@ public static Status GetStatus()
         {
             ServiceControllerStatus status;
 
-            using (var service = GetService()) {
-                if (service != null) {
-                    status = service.Status;
-                }
-                else {
-                    return Status.Unknown;
+            try
```

Comment:
Please check this again to figure out the root cause. Ignoring exception can be dangerous.